src/components/Pagination/Pagination.tsx

```diff
@@ -140,34 +140,39 @@ const Pagination: React.FC<PaginationProps> = ({
     return (
         <Container>
             <ButtonsContainer>
-                {hasMultiplePageSizes && (
-                    <Box aria-label={ariaLabelSelectPageSizeContainer} position="absolute" left="0" width="4.5em">
-                        <SelectList
-                            options={pageSizes}
-                            onChange={onSelectPageSize}
-                            value={pageSizes.find(sizeOption => sizeOption.value === pageSize.toString())}
-                        />
-                    </Box>
-                )}
-                {size !== 'small' && (
-                    <IconButton aria-label={ariaLabelFirst} disabled={isFirstPage} onClick={onSkipBackward}>
-                        <BackwardLastIcon size="small" color="inherit" />
-                    </IconButton>
-                )}
+                <Box flex="1">
+                    {hasMultiplePageSizes && (
+                        <Box aria-label={ariaLabelSelectPageSizeContainer} minWidth="5em" maxWidth="5.5em">
```

Comment:
shouldn't we remove the `maxWidth`?

Reply:
I edited the description to explain why if we remove the maxWidth the inner box will grow together with the outer one, having the PageSize selector grow way too much:

<img width="1104" alt="image" src="https://github.com/freenowtech/wave/assets/153822795/a26f8479-556f-4ecd-886f-2303f02b5c9b">

Reply:
As we spoke in Slack, I changed it to have the `width="fit-content"` instead of the maxWidth ;)